Translate alpine.md via GitLocalize (#11)
v2rayA/v2raya.github.io: content/en/docs/prologue/installation/alpine.md

## Install V2Ray Core/ Xray Core

> {{&lt; alert icon="👉" text="If you have already installed the core, you can skip this section." /&gt;}}

Reviewer: ```suggestion
{{< alert icon="👉" text="If you have already installed the core, you can skip this section." />}}
```

You can use the `arch` command to query your platform architecture. For example, for the x86_64 architecture, you need to download the x64 version.

> ### Create service file

Reviewer: ```suggestion
### Create a service file
```

Save the file, and then give the file executable permissions.

> ### Install the iptables module and release the 2017 port

Reviewer: ```suggestion
### Install the iptables module and make the 2017 port accessible
```
